wordpress-mobile/WordPress-Android
# Feature/site creation minor issues

Fixes 
- font size of subtitles on each screen
- size of the skip button on the site info screen
- adds decelerate interpolator to SitePreview transition


To test:
- Click on My Site
- Click on Switch Site
- Click on plus sign icon
- Select a segment
- Click on skip button
- notice the color of the skip button is white
- type something
- notice the color of the button changed to blue

![sub-titles](https://user-images.githubusercontent.com/2261188/50347557-352c9d80-0536-11e9-92d4-ad78a4243e3a.jpg)
![ef433501bd8bcf3d82195bb0f88a968c](https://user-images.githubusercontent.com/2261188/50347621-7755df00-0536-11e9-898d-56ee13eb947c.gif)

cc @SylvesterWilmott
